src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/resource/TestController.kt

```diff
@@ -18,14 +18,15 @@ import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.service.TestService
 class TestController(private val testService: TestService) {
 
   @GetMapping(value = ["/data"])
-  @PreAuthorize("hasAnyRole('SYSTEM_USER', 'ROLE_CALCULATE_SENTENCE_ADMIN')")
+  @PreAuthorize("hasAnyRole('SYSTEM_USER', 'CRD_ADMIN', 'PRISON')")
```

Comment:
Ultimately the the role will probably be SENTENCE_CALCULATOR, however at the moment just making this consistent with what's currently been set up in order to test the dev environment set up